src/store/slices/PlayersSectionsSlice.ts

```diff
@@ -171,13 +206,51 @@ export const {
   resetPlayersSections,
 } = PlayersSectionsSlice.actions;
 
-export const getDefaultSection = (state: IUIStoreState): number =>
+export const getDefaultSectionId = (state: IUIStoreState): number =>
   state.PlayersSections.defaultSection;
 
 export const getPlayersSections = (
   state: IUIStoreState
 ): Array<IPlayersSection> => state.PlayersSections.sections;
 
+export const getDefaultPlayersSection = (
+  state: IUIStoreState
+): IPlayersSection =>
+  _ensure(
+    state.PlayersSections.sections.find(
+      (playersSection) => playersSection.id === getDefaultSectionId(state)
+    )
+  );
+
+export const getDeadPlayersSection = (state: IUIStoreState): IPlayersSection =>
+  _ensure(
+    state.PlayersSections.sections.find(
+      (playerSection) =>
+        playerSection.id === state.PlayersSections.defaultDeadSection
+    )
+  );
+
+export const getUnusedPlayersSection = (
+  state: IUIStoreState
+): IPlayersSection =>
+  _ensure(
+    state.PlayersSections.sections.find(
+      (playerSection) =>
+        playerSection.id === state.PlayersSections.defaultUnusedSection
+    )
+  );
+
+// ensure Array.prototype.find will not return `undefined`
+function _ensure(
+  argument: IPlayersSection | undefined,
+  message = "This value was promised to be there."
+): IPlayersSection {
+  if (argument === undefined || argument === null) {
+    throw new TypeError(message);
+  }
+
+  return argument;
+}
```

Comment:
Seems like a function that can be transformed into a more generic utility: ```function _ensure<T>(argument: T | undefined): T {...}```

Reply:
yeah it's meant to be. i put an underscore in the function name to indicate internal function.

not sure how to write it in TypeScript. i'll update it